### ISSUE TITLE
Error in download_samples

### ISSUE THREAD

**Issue by CJ-Wright** [diffpy/sampledb#55]
It seems that the QR starts with the letter `e` then that `e` is missing from the excel sheet.

**Comment by justcalamari** [comment]
Weird. Have you noticed this with any other characters?

**Comment by CJ-Wright** [comment]
Nope, just e (I have a local copy if you want to take a look at some point)
